tools/cldr-apps/src/org/unicode/cldr/web/MailSender.java

```diff
@@ -336,6 +336,8 @@ private Properties getProperties() {
 
     private int lastIdProcessed = -1; // spinner
 
+    // Fetch one (1) row of mail that needs to be processed. Process it, then cause a new task to be created
```

Comment:
this confused me (though I wrote it :) …so i added the comment. 

Under heavy load, should more than one be processed at a time?